src/js/stores/validators/AppFormValidators.js

```diff
@@ -60,7 +60,7 @@ const AppFormValidators = {
       }),
 
   cpus: (value) => !Util.isStringAndEmpty(value) &&
-    !!value.toString().match(/^[0-9\.]+$/),
+    !!value.toString().match(/^[0-9\.]+$/) && value >= 0.01,
```

Comment:
Aj, I think the API would fail anyways, if a string like "0.2" is submitted? Not sure..